config/initializers/rollbar.rb

```diff
@@ -0,0 +1,57 @@
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```

Comment:
Will this come out in a different bucket of errors in the Rollbar UI, or same as JS?  I suspect it'll be obvious which one it's coming from just never tried it before! :)

Reply:
Yeah we'll see!